packages/flutter/packages/notifee_platform_interface/lib/src/models/notification/ios/foreground_presentation_options.dart

```diff
@@ -16,11 +16,12 @@
  */
 
 class ForegroundPresentationOptions {
-  ForegroundPresentationOptions({bool? alert, bool? badge, bool? sound, bool? banner, bool? list})
+  ForegroundPresentationOptions(
+      {bool? alert, bool? badge, bool? sound, bool? banner, bool? list})
       : alert = alert ?? false,
         badge = badge ?? false,
-        sound = sound ?? false;
```

Comment:
Thank you so much! 🙏

Reply:
no problem. thanks for updating the flutter plugin!